process/factory/shard/preProcessorsContainerFactory_test.go

```diff
@@ -43,6 +47,7 @@ func TestNewPreProcessorsContainerFactory_NilShardCoordinator(t *testing.T) {
 		&testscommon.TxTypeHandlerMock{},
 		&testscommon.ScheduledTxsExecutionStub{},
 		&testscommon.ProcessedMiniBlocksTrackerStub{},
+		common.ChainRunTypeRegular,
```

Comment:
we need and args struct here :D another refactor.

Reply:
Done